### DISCUSSION TITLE
Version 0.64

### DISCUSSION THREAD

**Discussion by bbernhard** [bbernhard/signal-cli-rest-api#294] [Announcements]
* updated signal-cli to version 0.11.3

This is a pre-release intended for testing. In case you want to give it a try, use the following docker image: `bbernhard/signal-cli-rest-api:0.113-dev`. Any feedback is highly appreciated ;-)

<hr /><em>This discussion was created from the release <a href='https://github.com/bbernhard/signal-cli-rest-api/releases/tag/0.64-pre'>Version 0.64</a>.</em>

**Comment by rfrederick** [comment]
It might be worth waiting for 0.11.4 as I ran into this blocker for me with 0.11.3: AsamK/signal-cli#1040. This prevented me from creating a profile for an account to be able to join it to a group until I switched to a signal-cli install built from the commit incorporating the fix.

**Comment by michaelkebe** [comment]
Just updated to the latest-dev docker image. No problems so far. But my setup is pretty simple - sending text message just to a single number.
